packages/shared/src/components/widgets/PostUsersHighlights.tsx

```diff
@@ -31,18 +32,14 @@ enum UserType {
 
 const StyledImage = classed(LazyImage, 'w-10 h-10');
 
-interface SourceAuthorProps {
-  id?: string;
-  handle?: string;
-  image: string;
-  name: string;
-  username?: string;
-  userType?: UserType;
-  permalink: string;
-  reputation?: number;
-}
```

Comment:
Needed to switch from `interface` to `type` for managing cases with dynamic props.